nni/compression/pytorch/compressor.py

```diff
@@ -63,7 +88,7 @@ def validate_config(self, model, config_list):
         """
         pass
 
-    def reset(self, checkpoint=None):
+    def reset_bound_model(self, checkpoint=None):
```

Comment:
what is this function used for?

Reply:
this function is used for re-wrap the model, used in `AMCPruner` and `reconfig()`